Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp

```diff
@@ -232,28 +221,25 @@ namespace AssetProcessor
             for (size_t optionIndex = 0; optionIndex < commandOptionCount; ++optionIndex)
             {
                 const AZStd::string& optionValue = commandLine.GetSwitchValue(optionKey, optionIndex);
-                params.append(AZStd::string::format(
-#if !AZ_TRAIT_OS_PLATFORM_APPLE && !AZ_TRAIT_OS_USE_WINDOWS_FILE_PATHS
-                    R"( --%s="%s")",
-#else
-                    R"( --%s="\"%s\"")",
-#endif
-                    optionKey, optionValue.c_str()));
+                params.emplace_back(AZStd::string::format(R"( --%s="%s")", optionKey, optionValue.c_str()));
```

Comment:
Since the arguments are added to individual vector elements the leading space before the `--` is no longer needed.

```suggestion
                params.emplace_back(AZStd::string::format(R"(--%s="%s")", optionKey, optionValue.c_str()));
```